ipod-shuffle-4g.py

```diff
@@ -124,6 +124,14 @@ def check_support():
         else:
             voiceoverAvailable = True
 
+        # Check for gtts-cli voiceover
+        # https://github.com/pndurette/gTTS
+        if not exec_exists_in_path("gtts-cli"):
```

Comment:
oh wait, we need to check for ffmpeg here as well, right?

Reply:
Dont we also need to check for `sox` of `rhvoice`? This is not related to this PR, but I just found this there as well.